devicon.json

```diff
@@ -567,6 +610,9 @@
     },
     {
         "name": "bash",
+        "altnames": [
+            "bash"
```

Comment:
Altname should not be allowed to be the same name as 'name' ;-) Need a check to adress this as well imo.

Reply:
Agreed. Not sure how to enforce it though. Maybe through github actions?